jetty-server/src/main/java/org/eclipse/jetty/server/CookieCutter.java

```diff
@@ -346,6 +369,20 @@ else if (tokenstart >= 0)
                                     quoted = false;
                                     continue;
                                 }
+
+                                if (_compliance == CookieCompliance.RFC6265)
+                                {
+                                    // Rejected cookie-octet characters
+                                    // US-ASCII characters excluding CTLs,
+                                    // whitespace DQUOTE, comma, semicolon,
+                                    // and backslash
+                                    if (Character.isISOControl(c) || Character.isWhitespace(c) ||
+                                        c == ',' || c == '\\')
+                                    {
+                                        reject = true;
```

Comment:
ditto as above

Reply:
This has been moved to a `isRFC6265RejectedCharacter(char c)` method later in the class.
Only Control characters are tested for now.

Reply:
I think we can be tougher if we are not in dquotes - so perhaps pass in `inQuotes` and check for tspecials only if not quoted.

Reply:
Will do.